github/downloader_test.go

```diff
@@ -151,3 +164,107 @@ func TestOnlineOrganizationDownload(t *testing.T) {
 	}
 
 }
+
+func loadReqResp(filepath string, reqResp map[string]string) error {
+	// Open a RO file
+	decodeFile, err := os.Open(filepath)
+	if err != nil {
+		return err
+	}
+	defer decodeFile.Close()
+	// Create a decoder
+	decoder := gob.NewDecoder(decodeFile)
+	// Decode
+	decoder.Decode(&reqResp)
+	return nil
+}
+
+// RoundTripFunc .
+type RoundTripFunc func(req *http.Request) *http.Response
+
+// RoundTrip .
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
+
+func TestOfflineOrganizationDownload(t *testing.T) {
+	var (
+		reqResp map[string]string = make(map[string]string)
```

Comment:
usually people write
```
reqResp := make(map[string]string)
```
in such case but up to you

Reply:
![image](https://user-images.githubusercontent.com/10910699/66501577-544bea00-eacc-11e9-8079-1bb299cb87a9.png)

@smacker is right